packages/showcases/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,289 @@
+import React, {
```

Comment:
This file is pretty big. Could you refactor this into separate components and hook?